packages/next/src/build/webpack/plugins/next-trace-entrypoints-plugin.ts

```diff
@@ -260,22 +289,60 @@ export class TraceEntryPointsPlugin implements webpack.WebpackPluginInstance {
         }
 
         const finalFiles: string[] = []
-
-        for (const file of new Set([
-          ...entryFiles,
-          ...(this.entryTraces.get(entrypoint.name) || []),
-        ])) {
-          if (file) {
-            finalFiles.push(
-              nodePath.relative(traceOutputPath, file).replace(/\\/g, '/')
-            )
-          }
+        const readFile = (path: string): Promise<Buffer | string> => {
+          return new Promise((resolve, reject) => {
+            compilation.inputFileSystem.readFile(path, (err, result) => {
+              if (err) {
+                return reject(err)
+              }
+              resolve(result || '')
```

Comment:
should we treat a missing result as an error too or is it ok for this to potentially return an empty string hash?

Reply:
I took this as an empty file which is technically valid so don't think we should error